<launch>
    <param name="name" value="yoon"/>
    <node pkg="tcps_image_attack" type="node_car_airsim.py" name="car_airsim_node" output="screen">
    </node>
    <node pkg="tcps_image_attack" type="node_yolo.py" name="yolo_node" output="screen">
    </node>
    <node pkg="tcps_image_attack" type="node_car_controller.py" name="car_controller_node" output="screen">
    </node>
</launch>
